### DISCUSSION TITLE
Github Projects and Issues config

### DISCUSSION THREAD

**Discussion by starkmobiledev** [community/community#109380] [Projects and Issues] [answered]
### Select Topic Area

Question

### Feature Area

Issues

### Body

How to config issue to automatically go to done after a date

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by KronosCodeur** [comment, marked as answer]
I think if you have a Github pro account you can config it.

**Reply by starkmobiledev** [reply]
thanks

**Reply by KronosCodeur** [reply]
i will check it with an classic account and back to it

**Comment by starkmobiledev** [comment]
can i config it without a Github pro account?

**Comment by KronosCodeur** [comment]
I think  you can do it 
it the workflow you can define the  some actions

**Comment by KronosCodeur** [comment]
i am going to check it
